### PR TITLE
LPFG-933: Manage organisations

### PR DESCRIPTION
**Pull request by m24murray**
This PR closes two preceding PRs, so basically has all of manage organisations in it 🧁

- organistion overview 
- edit organisations
- remove organisations
- updated service

### CODE REVIEW

File: src/controllers/organisationController.ts [hunk 1]
```diff
@@ -7,62 +7,159 @@ import {OrganisationalUnitFactory} from '../csrs/model/organisationalUnitFactory
...
 		}
 	}
 
-	public addOrganisation() {
+	public addEditOrganisation() {
 		return async (request: Request, response: Response) => {
 			const organisationalUnits: DefaultPageResults<OrganisationalUnit> = await this.csrs.listOrganisationalUnitsForTypehead()
 
-			response.render('page/organisation/add-organisation', {organisationalUnits: organisationalUnits})
+			response.render('page/organisation/add-edit-organisation', {organisationalUnits: organisationalUnits})
```
[the remark this answers is not in the document]
Comment (m24murray) [reply src/controllers/organisationController.ts]: Tbh, I'm really not a fan of getting templates by variable, theres not a whole lot to be gained from it and it prevents from doing text searches in codebase.

I would be happy to rename the file to something else if you feel strongly.

[the remark this answers is not in the document]
Comment (elxsy) [reply src/controllers/organisationController.ts]: I am talking about this part.

```
response.render('page/organisation/add-edit-organisation', {organisationalUnits: organisationalUnits})
```

```suggestion
			response.render('page/organisation/add-edit-organisation', organisationalUnits)
```

File: src/controllers/organisationController.ts [hunk 2]
```diff
@@ -7,62 +7,159 @@ import {OrganisationalUnitFactory} from '../csrs/model/organisationalUnitFactory
...
 
 			response.render('page/organisation/manage-organisations', {organisationalUnits: organisationalUnits})
 		}
 	}
 
-	public getOrganisation() {
+	public organisationOverview() {
 		return async (request: Request, response: Response) => {
-			response.render('/content-management/organisations')
+			const organisationalUnit = response.locals.organisationalUnit
```
[the remark this answers is not in the document]
Comment (m24murray) [reply src/controllers/organisationController.ts]: Good catch, this is just an old version of the method. Have removed

File: src/controllers/organisationController.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (m24murray) [reply src/controllers/organisationController.ts]: Ah my bad, yeah sure changes made

**Pull request merged**